### PR TITLE
Update changelog for 2.3.1

### PR DESCRIPTION
**Pull request by ph**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+## 2.3.1 (Apr 4, 2016)
+### general
+ - Fix a JRuby thread safety issue when using regular expression under multiple workers ([#4977](https://github.com/elastic/logstash/issues/4977)).
```
Comment (colinsurprenant) [inline CHANGELOG.md]: in prior logs we use the first vern in the past tense "fixed", "disabled", ... we should be consistent

**Comment by colinsurprenant** [comment]
Other than my inline comment on the wording consistency, LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (ph) [inline CHANGELOG.md]: agreed,OOPs

**Comment by ph** [comment]
fixed

**Comment by ph** [comment]
commit https://github.com/elastic/logstash/commit/3ab541452d817e98c13824e7aab2b385b1fdeab8

**Pull request closed without merging**